test/integration.test.ts

```diff
@@ -59,32 +58,51 @@ function assertValid(
   }
 }
 
-describe("GET endpoints", async function () {
-  const map: { path: PathsWithMethod<paths, "get">; id: string }[] = [
-    { path: "/status", id: "NodeStatus" },
-    { path: "/controller", id: "ControllerStatus" },
-    { path: "/network", id: "JoinedNetworks" },
-    { path: "/peer", id: "Peers" },
-  ];
+const ZEROTIER_API_SECRET = "asdf";
 
-  for (const { path, id } of map) {
-    it(id, async () => {
-      const validator = createValidator(id);
+describe("API exercise", async function() {
+  let network_id: string;
+  const node_id = "1122334455";
+  let container;
+  let client: ReturnType<typeof createCreateClient>;
+  let apiPort: number;
+
+  before(async () => {
+    container = await new GenericContainer(`zerotier/zerotier:${ZT_VERSION}`)
+      .withExposedPorts(9993)
+      .withPrivilegedMode()
+      .withAddedCapabilities("NET_ADMIN")
+      .withEnvironment({ ZEROTIER_API_SECRET })
+      .withCopyContentToContainer([
+        {
+          content: `{ "settings": { "allowManagementFrom": ["0.0.0.0/0"] } }`,
```

Comment:
Nitpick: if we use JSON.stringify on a native JS object for injecting this kind of configuration (vs. string templates) we could pretty easily make the specific config you pass overridable/composable.

E.g.:

```
content: JSON.stringify({ ...DEFAULT_LOCAL_CONF, allowManagementFrom: [ "0.0.0.0/0" ]})
```

(Again, just a stylistic thing, and totally fine to ignore until we actually have >1 config file body to populate + inject.)

Reply:
just a hack to get it to work as expected in docker for mac